glusterd2/commands/volumes/volume-options-get.go

```diff
@@ -0,0 +1,104 @@
+package volumecommands
+
+import (
+	"net/http"
+
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/glusterd2/xlator/options"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/errors"
+
+	"github.com/gorilla/mux"
+)
+
+func volumeOptionsGetHandler(w http.ResponseWriter, r *http.Request) {
+
+	ctx := r.Context()
+
+	volname := mux.Vars(r)["volname"]
+
+	v, err := volume.GetVolume(volname)
+	if err != nil {
+		if err == errors.ErrVolNotFound {
+			restutils.SendHTTPError(ctx, w, http.StatusNotFound, err)
```

Comment:
HTTP status is confusing here. Option not found vs Volume not found. I think `StatusInternalServerError` itself can be used here. Thoughts?

Reply:
the volume name is an user input, I fell Not found is the best-suited option if volume not found in ETCD
```
status, err := restutils.ErrToStatusCode(err)
```
This function returns back the status code based on the error.